learning/courses/basics-of-quantum-information/entanglement-in-action/introduction.ipynb

```diff
@@ -7,7 +7,10 @@
    "source": [
     "# Introduction\n",
     "\n",
-    "Click on the following image to access the lesson video.\n",
+    "In the video below John Watrous steps you through the content in this lesson on entanglement in action. Alternatively, you can open the [YouTube video](https://youtu.be/GSsElSQgMbU?list=PLOFEBzvs-VvqKKMXX4vbi4EB1uaErFMSO) for this lesson in a separate window.\n",
+    "\n",
+    "<IBMVideo id=\"134056223\" title=\"In this video John Watrous introduces quantum entanglement. He further demonstrates how entanglement is leveraged in so-called quantum teleportation. Finally he describes how entanglement enables superdense coding.\"/>\n",
+    "\n",
     "\n",
     "<ImageLink\n",
```

Comment:
You'll also want to remove the ImageLink. It's going to want you to as well delete the corresponding image. (If you're busy on other tasks other people cannot do, it may be worth asking for help with this task)